src/components/layer-controller/ColorPalette.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import LensIcon from '@material-ui/icons/Lens';
+import { makeStyles } from '@material-ui/core/styles';
+
+
+import { VIEWER_PALETTE } from '../utils';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    width: '70px',
+    height: '40px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+  },
+  button: {
+    padding: '3px',
+    width: '16px',
+    height: '16px',
+  },
+  icon: {
+    width: '17px',
+    height: '17px',
+  },
+}));
+
+const ColorPalette = ({ handleChange }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.container} aria-label="color-swatch">
+      {VIEWER_PALETTE.map(color => (
+        <IconButton
+          className={classes.button}
+          key={color}
+          onClick={() => handleChange(color)}
+        >
+          <LensIcon
+            fontSize="small"
```

Comment:
Yes, I believe in style "fontSize" would require a pixel or em value, but material adds these as props for components so it will use "small" as defined be the theme. At least this is my understanding.

Reply:
This type of magic and inconsistencies with when to use in-line styles or `makeStyles` is why I'd like to find a better CSS solution that we can all adhere to.